acl/src/test/resources/conf/plain_acl/plain_acl_default.yml

```diff
@@ -0,0 +1,20 @@
+globalWhiteRemoteAddresses:
```

Comment:
Need to add Licensed descrption, you can refer my original config yml files.

Reply:
done